src/ts/component/editor/page.tsx

```diff
@@ -793,10 +793,18 @@ const EditorPage = observer(class EditorPage extends React.Component<Props, Stat
 			});
 
 			if (block.isTextToggle()) {
-				keyboard.shortcut(`${cmd}+shift+t`, e, () => {
```

Comment:
Changing the shortcut itself should be more of a feature request, I can't allow this without a product review.

Reply:
I thought that might be the case. I'll revert back.